festim/hydrogen_transport_problem.py

```diff
@@ -155,7 +202,7 @@ def define_markers_and_measures(self):
 
         for sub_dom in self.subdomains:
             if isinstance(sub_dom, F.SurfaceSubdomain1D):
-                dof = sub_dom.locate_dof(self.function_space)
+                dof = sub_dom.locate_dof(self.mesh.mesh, self.mesh.fdim)
```

Comment:
See another comment below but I don't know if this is actually a dof or a facet index